packages/xo-web/src/xo-app/backup/new/smart-backup.js

```diff
@@ -29,7 +31,31 @@ const SmartBackup = decorate([
     vms: createGetObjectsOfType('VM'),
   }),
   provideState({
+    initialState: () => ({
+      editingTag: false,
+    }),
     effects: {
+      addTag: (effects, newTag) => ({ tags }) => {
+        effects.setTagValues(
+          tags.values === undefined ? [newTag] : [...tags.values, newTag]
+        )
+      },
+      onKeyDown: (effects, event) => {
+        const { keyCode, target } = event
+
+        if (keyCode === 13) {
+          if (target.value !== '') {
+            effects.addTag(target.value)
+            target.value = ''
+          }
+        } else if (keyCode === 27) {
+          effects.stopEditTag()
+        } else {
+          return
+        }
+
+        event.preventDefault()
```

Comment:
I think that `event.preventDefault()` should come in the begining of this effect